README.md

```diff
@@ -13,4 +13,8 @@ This code of conduct applies both within project spaces and in public spaces whe
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by opening an issue or contacting one or more of the project maintainers.
 
+Unpopular opinions are welcome, so long as their assumptions do not challenge those of the mission.  For example, opinions like racial superiority are unacceptable because they necessary conflict with egalitarianism and universal human rights, which is the foundation of our mission of inclusivity.  On the other hand, conservative, libertarian, and other non-progressive opinions are acceptable so long as they are not predicated on any assumptions that conflict with the moral imperatives of modern, Western society: equality, democracy, and non-violence.
```

Comment:
I'd suggest removing the terms "modern, Western society:" from this. equality, democracy, and non-violence are not limited to modern society, nor western society. Historicism aside, I'm not sure what they add to this point.

Reply:
Updated.  You're right - there's no reason to bring history into this.